index.html

```diff
@@ -0,0 +1,199 @@
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="row join-container d-md-none p-3 mt-3 mb-3 justify-content-center">
+          <div class="col-10 join-bg pe-4 ps-4 pt-3 pb-3 d-flex justify-content-center">
+            <p class="mb-0 fw-bold">GET YOUR TICKETS NOW!</p>
+          </div>
+        </div>
+
+        <div class="row p-md-5 see-program">
+          <div class="col d-flex justify-content-center">
+            <p class="border-bottom border-2 border-secondary">SEE THE WHOLE PROGRAM</p>
+          </div>
+        </div>
+      </div>
+    </main>
+  </section>
+  <article>
+    <div class="container-fluid ps-md-5 pe-md-5" id="feature-speaker">
+      <div class="d-flex flex-column justify-content-center align-items-center pt-4 pb-4">
+        <p class="mb-2 fw-bolder fs-5">Featured Speaker</p>
+        <div class="orange-line"></div>
+      </div>
+    </div>
+  </article>
+  <footer>
+    <div class="container-fluid partner-sec pt-md-4 pb-md-4">
+      <div class="d-flex flex-column justify-content-center align-items-center pt-4 pb-4">
+        <p class="mb-2 fw-bolder fs-5 partner-title">Partner</p>
+        <div class="orange-line orange-line-desk"></div>
+      </div>
+      <div class="row row-cols-3 row-cols-md-auto partner-sec-font justify-content-center">
+        <div class="col d-flex justify-content-center align-items-center"><p class=" text-light-gray font-style-1">Y<span class="text-dark-gray">o</span>uT<span class="text-dark-gray">ub</span>e</p></div>
+        <div class="col d-flex justify-content-center align-items-center"><p class="text-light-gray font-style-2">Spotify</p></div>
+        <div class="col d-flex justify-content-center align-items-center"><p class="text-light-gray font-style-4 pt-2">Facebook</p></div>
+        <div class="col d-flex justify-content-center align-items-center"><p class="text-light-gray font-style-3 text-center">Google Podcasts</p></div>
+        <div class="col d-flex justify-content-center align-items-center"><p class="text-light-gray font-style-5">SoundCloud</p></div>
+      </div>
+    </div>
+    <div class="container-fluid">
+      <div class="row footer-container justify-content-center align-items-center mt-4 mb-4 p-md-3">
+        <div class="col-6 d-flex col-md-3 justify-content-md-end justify-content-center footer-logo align-items-center mb-0">
+          <img src="img/logo.png" class="partner-logo">
+        </div>
+        <div class="col-6 ps-0 d-md-flex justify-content-md-center">
+          <p class="footer-desc mb-0">2023 Supreme Media.All Rights Reserved. <br> Contact us via email @supremesummit2023@myanmar.org</p>
+        </div>
+      </div>
+    </div>
+  </footer>
```

Comment:
Please, Let's try to stick to the design for the mobile section of your index page😉. It shouldn't have a footer and it should end with the see more button😉.

yours
<img width="299" alt="image" src="https://github.com/K0ppai/first-capstone-project/assets/64643995/54a6a6b5-66e1-4275-968d-afcd41ac3344">
design
<img width="299" alt="image" src="https://github.com/K0ppai/first-capstone-project/assets/64643995/ef3f8ca4-16f4-4aa4-9f4d-2ffe573b135f">